pkg/registry/csv.go

```diff
@@ -58,6 +58,9 @@ const (
 
 	// The yaml attribute that specifies the skipRange of the ClusterServiceVersion
 	skipRangeAnnotationKey = "olm.skipRange"
+
+	// The yaml attibure that specifies the optional substitutesfor of the ClusterServiceVersion
+	substitutesForAnnotationKey = "substitutesFor"
```

Comment:
if this is for an annotation it should be namespaced - we have used `olm` for skiprange